shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -142,9 +142,12 @@
       <li><button type="button" class="login-dev-account" data-name="Dave Dev User">
           Dave
       </button></li>
-      <li><button type="button" class="login-dev-account" data-name="Eve Dev User">
-          Eve
-      </button></li>
+      <li>
+        <div class="flex-box">
+          <input name="name" class="custom-name" type="text" placeholder="Eve" required>
```

Comment:
Right, the other buttons have click handlers. It would perhaps be nicer for them not to be part of the form element, but then we would have to rejigger the styling, which currently benefits from closely following the pattern of the the email login form.